src/java.desktop/share/classes/java/awt/FlowLayout.java

```diff
@@ -51,31 +51,38 @@
  * <li>{@link #TRAILING TRAILING}
  * </ul>
  * <p>
- * For example, the following picture shows an applet using the flow
+ * For example, the following picture shows a window using the flow
  * layout manager (its default layout manager) to position three buttons:
  * <p>
- * <img src="doc-files/FlowLayout-1.gif"
+ * <img src="doc-files/FlowLayout-1.png"
  * ALT="Graphic of Layout for Three Buttons"
  * style="margin: 7px 10px;">
  * <p>
- * Here is the code for this applet:
+ * Here is the code for this program:
  *
- * <hr><blockquote><pre>
- * import java.awt.*;
- * import java.applet.Applet;
+ * <hr>
+ * {@snippet  lang='java':
+ * import java.awt.Button;
+ * import java.awt.EventQueue;
+ * import java.awt.FlowLayout;
+ * import java.awt.Frame;
  *
- * public class myButtons extends Applet {
- *     Button button1, button2, button3;
- *     public void init() {
- *         button1 = new Button("Ok");
- *         button2 = new Button("Open");
- *         button3 = new Button("Close");
- *         add(button1);
- *         add(button2);
- *         add(button3);
- *     }
+ * public class FlowLayoutExample {
+ *
+ *   public static void main(String[] args) throws Exception {
+ *
+ *     EventQueue.invokeAndWait(() -> {
+ *        Frame frame = new Frame("FlowLayout");
```

Comment:
The same comment: let's follow the standard Java code style and indent blocks by 4 spaces.

Here, the `main` method declaration and `EventQueue.invokeAndWait` are indented by 2 spaces, but the body of the lambda expression is indented by 4 spaces. This is inconsistent.